src/pages/storage/StorageUsedBy.tsx

```diff
@@ -36,10 +34,13 @@ const StorageUsedBy: FC<Props> = ({ storage, project }) => {
           <td>
             <ExpandableList
               items={data[INSTANCES].map((item) => (
-                <div key={`${item.name}-${item.project}`}>
-                  <InstanceLink instance={item} />
-                  {item.project !== project && ` (project ${item.project})`}
-                </div>
+                <UsedByItem
+                  key={`${item.name}-${item.project}`}
+                  item={item}
+                  project={project}
+                  type="virtual-machine"
```

Comment:
Can be a container or vm.

Reply:
related to comments from #946, we can use a general `instance` type to generate the link here. Alternatively, we could also make an API call at this point to get instance type detail. Prefer the first approach as it's simple and less expensive